docs/sql-ref-syntax-aux-show-tables.md

```diff
@@ -18,5 +18,83 @@ license: |
   See the License for the specific language governing permissions and
   limitations under the License.
 ---
+### Description
 
-**This page is under construction**
+List all the `tables` from the database with `database-name` and `is temporary table`.
```

Comment:
@shivusondur Sorry.. re-reading this one last time :-)
`SHOW TABLES` statement returns all the tables for an optionally specified database. Additionally, the output
of this statement may be filtered via an optional matching pattern. If no database is specified then the tables
are returned from the current database.

Reply:
DOne